driver-core/src/main/java/com/datastax/driver/core/Metrics.java

```diff
@@ -219,6 +229,15 @@ public Errors getErrorMetrics() {
     }
 
     /**
+     * Returns the total number of in flight requests to Cassandra hosts.
+     *
+     * @return The total number of in flight requests to Cassandra hosts.
+     */
+    public Gauge<Integer> getInFlightRequests() {
```

Comment:
It's a good suggestion that several folks actually prefer as well on our side here! Let me update the PR accordingly.